spec/pactas_itero/client_spec.rb

```diff
@@ -301,7 +301,7 @@
     it 'returns url of the sandbox endpoint' do
       client = PactasItero::Client.new
 
-      expect(client.sandbox_api_endpoint).to eq 'https://sandbox.pactas.com'
+      expect(client.sandbox_api_endpoint).to eq 'https://sandbox.billwerk.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.